### PR TITLE
Automated Tests - CommerceProducts - combined PRs

### PR DESCRIPTION
**Pull request by pedro-michel**

### CODE REVIEW

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by pedro-michel** [comment]
https://issues.liferay.com/browse/COMMERCE-5802
https://issues.liferay.com/browse/COMMERCE-5805
https://issues.liferay.com/browse/COMMERCE-5530
https://issues.liferay.com/browse/COMMERCE-5842
https://issues.liferay.com/browse/COMMERCE-5843
https://issues.liferay.com/browse/COMMERCE-5845
https://issues.liferay.com/browse/COMMERCE-5859
https://issues.liferay.com/browse/COMMERCE-5860
https://issues.liferay.com/browse/COMMERCE-5862
https://issues.liferay.com/browse/COMMERCE-5864
https://issues.liferay.com/browse/COMMERCE-5982
https://issues.liferay.com/browse/COMMERCE-6023

**Comment by kristinonias** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 1]
```diff
@@ -724,22 +822,45 @@ definition {
...
+		AssertElementPresent(locator1 = "//a[contains(text(),'U-Joint')]");
+
+		AssertTextEquals(
+			locator1 = "Pagination#RESULTS_2",
+			value1 = "Showing 1 to 2 of 2");
 	}
 
-	@description = "This is a test for COMMERCE-5806. UseSkuSearchBarAndPagination"
+	@description = "This is a test for COMMERCE-5801. UseSkuSearchBarAndPagination"
+	@ignore = "true"
```
Comment (brittneyq) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:858]: please remove @ignore = "true"

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by kristinonias** [comment]
ci:test:commerce-functional

**Comment by kristinonias** [comment]
https://issues.liferay.com/browse/COMMERCE-6023

**com.liferay.poshi.runner.exception.ElementNotFoundPoshiRunnerException: Element is not present at "xpath=(//div | //li)[(contains(@class,'open') or contains(@class,'show')) and (contains(@class,'dropdown') or contains(.//*/@class,'dropdown'))]//*[self::a or self::span or self::button][normalize-space(text())='Master']"**

https://issues.liferay.com/browse/COMMERCE-5845

**java.lang.Exception: Element is not present at "//div[@id='main-content']//div[contains(@class,'portlet')]//section[contains(@id,'Checkout')]"**

**Comment by kristinonias** [comment]
https://issues.liferay.com/browse/LRQA-67149

**Comment by pedro-michel** [comment]
New PR Sent : https://github.com/liferay-commerce/liferay-portal/pull/1469

**Pull request closed without merging**